app/models/highlight_suggestion.rb

```diff
@@ -42,7 +44,7 @@ def from_run(run)
             end
           )
           # 60 days is life of archives for Partners / Twitch Prime members
-          highlight_suggestion.delay(run_at: video_start + 60.days).destroy if highlight_suggestion.persisted?
+          HighlightCleanupJob.set(wait: 60.days).perform_later(highlight_suggestion) if highlight_suggestion.persisted?
```

Comment:
Any simple way to make this 60 days from video start and not 60 days from upload?